### ISSUE TITLE
Issue running tests

### ISSUE THREAD

**Issue by mboisson** [easybuilders/easybuild-framework#2659]
```
[mboisson@build-node easybuild-framework]$ python -O -m test.framework.suite test_update
INFO: This is (based on) vsc.install.shared_setup 0.10.17
Traceback (most recent call last):
  File "/cvmfs/soft.computecanada.ca/nix/store/4x0hqnpd0hfh62m3apkxmz8hz3hlsikx-python-2.7.13-env/lib/python2.7/runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "/cvmfs/soft.computecanada.ca/nix/store/4x0hqnpd0hfh62m3apkxmz8hz3hlsikx-python-2.7.13-env/lib/python2.7/runpy.py", line 72, in _run_code
    exec code in run_globals
  File "/home/mboisson/git/easybuild-framework/test/framework/suite.py", line 44, in <module>
    import test.framework.asyncprocess as a
  File "/home/mboisson/git/easybuild-framework/test/framework/asyncprocess.py", line 34, in <module>
    from test.framework.utilities import EnhancedTestCase
  File "/home/mboisson/git/easybuild-framework/test/framework/utilities.py", line 71, in <module>
    go = EasyBuildOptions(go_useconfigfiles=False)
  File "easybuild/tools/options.py", line 246, in __init__
    super(EasyBuildOptions, self).__init__(*args, **kwargs)
  File "/cvmfs/soft.computecanada.ca/nix/store/4x0hqnpd0hfh62m3apkxmz8hz3hlsikx-python-2.7.13-env/lib/python2.7/site-packages/vsc/utils/generaloption.py", line 982, in __init__
    self.main_options()
  File "/cvmfs/soft.computecanada.ca/nix/store/4x0hqnpd0hfh62m3apkxmz8hz3hlsikx-python-2.7.13-env/lib/python2.7/site-packages/vsc/utils/generaloption.py", line 1058, in main_options
    fn()
  File "easybuild/tools/options.py", line 493, in config_options
    'choice', 'store', DEFAULT_REPOSITORY, sorted(avail_repositories().keys())),
  File "easybuild/tools/repository/repository.py", line 152, in avail_repositories
    raise EasyBuildError("avail_repositories: FileRepository missing from list of repositories")
easybuild.tools.build_log.EasyBuildError: 'avail_repositories: FileRepository missing from list of repositories'
``` 

Opening an issue for @boegel

**Comment by bartoldeman** [comment]
Weirdly I do not see the same issue on the same node. I have run various test cases there many times.

**Comment by mboisson** [comment]
Maybe it's just a config in my account then.

**Comment by bartoldeman** [comment]
So basically for some reason I don't fully understand (I probably did something related to setuptools in January) my git checkout has a `easybuild_framework.egg-info` directory but Maxime has not. Once we copied it across, the tests worked.

**Comment by boegel** [comment]
How was this `easybuild-framework` installed done exactly?

**Comment by mboisson** [comment]
`git clone git@github.com:ComputeCanada/easybuild-framework.git` 
(and added easybuilders as a second remote)

**Comment by boegel** [comment]
@mboisson There must be more to it... What about `vsc-base`? How is that picked up? `$PYTHONPATH`?

**Comment by boegel** [comment]
@victorusu was running into the same error message. In his case it happened because i) he was running the tests from the `easybuild-framework` directory, ii) he had a `setuptools.pth` available in `$PYTHONPATH` which seems to have as a side-effect that the current working directory is not included in the Python search path (`sys.path`).

**Comment by boegel** [comment]
See #2742 for more details on the specific problem that @victorusu was hitting, could be a duplicate of this problem?

**Comment by boegel** [comment]
problem should be fixed with changes in #2786
